package.json

```diff
@@ -35,7 +35,8 @@
   "files": [
     "bin",
     "lib/*.js",
-    "grammars/links.js",
+    "promise.js",
```

Comment:
Packaging is hard to test, but when I was publishing the new version to NPM I noticed this still isn't correct - it should be `promises.js`

Reply:
Fixed https://github.com/jamesramsay/hercule/pull/621